packages/vuetify/src/components/VStepper/VStepperStep.ts

```diff
@@ -104,6 +104,7 @@ export default baseMixins.extend<options>().extend({
     genLabel () {
       return this.$createElement('div', {
         staticClass: 'v-stepper__label',
+        attrs: { title: this.$slots.default && this.$slots.default.length > 0 ? this.$slots.default[0].text : null },
```

Comment:
We can pass attrs here, but no explicit title declaration. We should instead opt for a11y support. @phiter any feedback?

Reply:
I think it is valid to use title since it works for visually enabled people to check the full contents on hover.

`aria-label` would only work for screen readers, but we don't need that since even though the text is visually hidden, it is readable by the screen reader.

Reply:
I suppose the larger concern is assuming title without also passing in the props. I'd opt to pass the title attribute to it for now and let that be part of another PR since the scope of this is to fix the label.

Reply:
On that case I agree. The user should be able to pass the title. This change will always add a title to the labels even if they're small enough not to be collapsed.